generate_context_vectors.py

```diff
@@ -136,14 +97,10 @@ def main(
                 (section, _) = batched_tokens[j]
                 context_vectors.append((section, batch_context_vectors[j]))
 
-        # Construct the MongoDB document
         for section, context_vector in context_vectors:
-            document = {
-                "title": article["section_name"],
-                "context_vector": context_vector,
-                "header_name": section
-            }
-            collection.insert_one(document)
+            # TODO: section or article["section_name"]
+            # @nate please clarify
+            cv_db.insert(section, context_vector)
```

Comment:
@nalane what is the difference between `article["section_name"]` and `section`? Do we need both or is just `section` fine?